### PR TITLE
One Video Adapter

### PR DESCRIPTION
**Pull request by ankur-modi**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'oneVideo',
            params: {
              video: {
                playerWidth: 480,
                playerHeight: 640,
                mimes: ['video/mp4', 'application/javascript'],
                protocols: [2,5],
                api: [2],
                position: 1,
                delivery: [2]
              },
              site: {
                id: 1,
                page: 'http://abhi12345.com',
                referrer: 'http://abhi12345.com'
              },
              pubId: 'brxd'
            }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: ankur.modi@oath.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/
https://github.com/prebid/prebid.github.io/pull/739

### CODE REVIEW

**Comment by snapwich** [comment]
@ankur-modi can you please fill out the description of your submission to have proper params, docs, etc?

File: modules/oneVideoBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(bids) {
+    return bids.map(bid => {
+      console.log(getRequestData(bid));
```
Comment (snapwich) [inline modules/oneVideoBidAdapter.js]: please remove all debugging information.  there shouldn't be any `console.log` in the production code (other than through `utils.logWarn`, `utils.logError`, etc.

File: test/spec/modules/oneVideoBidAdapter_spec.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      const requests = spec.buildRequests([ bidRequest ]);
+      expect(requests[0].method).to.equal('POST');
+      expect(requests[0].url).to.equal(location.protocol + spec.ENDPOINT + bidRequest.params.pubId);
+    });
+
+    it('should attach the bid request object', () => {
+      const requests = spec.buildRequests([ bidRequest ]);
+      expect(requests[0].bidRequest).to.equal(bidRequest);
+    });
+  });
```
Comment (snapwich) [inline test/spec/modules/oneVideoBidAdapter_spec.js]: should also be validating that the data payload looks correct

File: modules/oneVideoBidAdapter.js (hunk 1 above)
Comment (ankur-modi) [reply modules/oneVideoBidAdapter.js]: done

File: test/spec/modules/oneVideoBidAdapter_spec.js (hunk 2 above)
Comment (ankur-modi) [reply test/spec/modules/oneVideoBidAdapter_spec.js]: done

**Pull request merged**